packages/vuetify/src/stylus/settings/_variables.styl

```diff
@@ -194,11 +194,13 @@ $card-border-radius := 2px
 // Chip
 $chip-border-radius := 28px
 $chip-close-margin := 0 2px 0 8px
+$chip-close-margin-rtl := 0 8px 0 2px
 $chip-font-size := 13px
 $chip-label-border-radius := $card-border-radius
 $chip-icon-font-size := 20px
 $chip-padding := 0 12px
 $chip-removeable-padding := 0 4px 0 12px
```

Comment:
Oh it was already wrong here

Reply:
I'd probably remove these and make it relate to `$chip-icon-offset` and `$chip-margin` instead.